.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 test/work
+.idea/
```

Comment:
If you don't mind, I want to add it — it is IDEA/WebStorm project files directory. In case of this project I don't want to add my IDEA project to VCS but want to ignore it.